k8s/prod/common/em/em-stitching.yaml

```diff
@@ -1,36 +0,0 @@
----
-apiVersion: helm.fluxcd.io/v1
-kind: HelmRelease
-metadata:
-  name: em-stitching
-  namespace: em
-  annotations:
-    flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
-spec:
-  releaseName: em-stitching
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
-    name: em-stitching
-    version: 1.0.0
-  values:
-    java:
-      replicas: 2
-      memoryLimits: "4096Mi"
-      cpuLimits: "2000m"
-      useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/em/stitching:prod-581d6db1
-      ingressHost: em-stitching-prod.service.core-compute-prod.internal
```

Comment:
You have ingress defined in here, but not in the prod patch. I cant see it in here as well. https://github.com/hmcts/em-stitching-api/blob/master/charts/em-stitching/values.yaml This will break things.

Reply:
I have added it in values.yaml now